packages/kbn-reporting/export_types_helpers_public/metrics.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { PdfScreenshotResult, PngScreenshotResult } from '@kbn/screenshotting-plugin/server';
+
+export type PngMetrics = PngScreenshotResult['metrics'];
+
+export type PdfMetrics = PdfScreenshotResult['metrics'];
+
+export interface CsvMetrics {
+  rows: number;
+}
+
+export interface TaskRunMetrics {
+  csv?: CsvMetrics;
+  png?: PngMetrics;
+  pdf?: PdfMetrics;
+}
+
+export interface TaskRunResult {
```

Comment:
This is exported from a few places. Search the code for `export interface TaskRunResult`